### PR TITLE
[DataGrid] Add esES locale

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

**Comment by DanailH** [comment]
@WiXSL Thanks for the contribution!

**Review by DanailH — APPROVED** [review]

File: packages/grid/_modules_/grid/locales/esES.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  // Column header text
+  columnHeaderFiltersTooltipActive: (count) =>
+    count !== 1 ? `${count} filtros activos` : `${count} filtro activo`,
+  columnHeaderFiltersLabel: 'Mostrar filtros',
+  columnHeaderSortIconLabel: 'Ordenar',
+
+  // Rows selected footer text
+  footerRowSelected: (count) =>
+    count !== 1
```
Comment (oliviertassinari) [inline packages/grid/_modules_/grid/locales/esES.ts:81]: Are you sure? As far as I remember my Spanish lessons, it's
```suggestion
    count > 1
```

Comment (WiXSL) [reply packages/grid/_modules_/grid/locales/esES.ts]: You are right!

Comment (WiXSL) [reply packages/grid/_modules_/grid/locales/esES.ts]: I just copied from the `enUs` locale

**Review by oliviertassinari — APPROVED** [review]

**Pull request merged**